disputer/disputer.js

```diff
@@ -74,13 +74,22 @@ class Disputer {
 
     // Get the latest disputable liquidations from the client.
     const undisputedLiquidations = this.empClient.getUndisputedLiquidations();
-    const disputeableLiquidations = undisputedLiquidations.filter(
-      liquidation =>
-        this.empClient.isDisputable(
-          liquidation,
-          this.priceFeed.getHistoricalPrice(parseInt(liquidation.liquidationTime.toString()))
-        ) && this.empClient.getLastUpdateTime() >= Number(liquidation.liquidationTime) + this.disputeDelay
-    );
+    const disputeableLiquidations = undisputedLiquidations.filter(liquidation => {
+      const price = this.priceFeed.getHistoricalPrice(parseInt(liquidation.liquidationTime.toString()));
+      if (!price) {
+        this.logger.warn({
+          at: "Disputer",
+          message: "Cannot dispute: price feed returned invalid value"
+        });
+      } else {
+        if (
+          this.empClient.isDisputable(liquidation, price) &&
+          this.empClient.getLastUpdateTime() >= Number(liquidation.liquidationTime) + this.disputeDelay
+        ) {
+          return liquidation;
```

Comment:
nit: I think the return value for a filter function is a boolean -- `true` if you want the element included and `false` if you don't.

Reply:
fixed this